client/lib/signup/step-actions.js

```diff
@@ -159,12 +159,17 @@ export function createSiteWithCart( callback, dependencies, stepData, reduxStore
 	// flowName isn't always passed in
 	const flowToCheck = flowName || lastKnownFlow;
 
+	// We will use the default annotation instead of theme annotation as fallback,
+	// when segment and vertical values are not sent. Check pbAok1-p2#comment-834.
+	const shouldUseDefaultAnnoationAsFallback = true;
```

Comment:
`shouldUseDefaultAnnotationAsFallback`

Reply:
Annoation still means annotation in Klingon.